src/components/product_detail/section1.js

```diff
@@ -0,0 +1,218 @@
+    font-family: Kanit;
+    color: #828282;
+    font-weight: normal;
+    margin-right: 25px;
+    margin-bottom: 20px;
+`
+
+const Price = styled.p`
+    font-family: Kanit;
+    font-size: 3.6rem;
+    width: 13.3rem;
+    color: #5b3c78;
+    line-height: 3.6rem;
+    margin-left: 0.8rem;
+    margin-bottom: 20px;
+`
+
+const Shipping = styled.p`
+    font-family: Kanit;
+    color: #828282;
+    font-weight: normal;
+    margin-right: 25px;
+    margin-bottom: 20px;
+`
+
+const Amount = styled.p`
+    font-family: Kanit;
+    color: #828282;
+    font-weight: normal;
+    margin-right: 45px;
+    margin-bottom: 40px;
+`
+
+const Container_PlusandMinus = styled.div`
+    display: flex;
+    flex-direction: row;
+    margin-right: 26px;
+`
+
+const ConntainerAdd_Buy = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+`
+
+const Button_Add = styled(Button)`
+    && {
+        display: flex;
+        flex-direction: row;
+        margin-right: 30px;
+        border: solid 1px #e89c6b;
+        padding: 10px;
+        height: 40px;
+        width: 179px;
+        background: rgba(232, 156, 107, 0.2);
+    }
+`
+
+const Detail_Text = styled.p`
+    font-family: Kanit;
+    color: #e89c6b;
+    font-size: 14px;
+    margin-right: 10px;
+`
+
+const  Detail_Cart = styled.img`
+    width: 23px;
+    height: 23px;
+    font-size: 14px;
+`
+
+
+const Button_Buy = styled(Button)`
+    && {
+        font-family: Kanit;
+        padding: 10px;
+        height: 40px;
+        width: 179px;
+        color: #f2f2f2;
+        background: #e89c6b;
+        font-size: 14px;
+    }
+`
+
+
+const Section1 = () => {
+    return (
+        <>
+            <Content>
+                <LeftItem>
+                    <Image/>
+                </LeftItem>
+                <RightItem>
+                    <ProductName>สับปะรดกับมะพร้าว</ProductName>
+                    <ContainerDetial>
+                        <ContainerStar>
+                            <StarRate/>
+                        </ContainerStar>
+                            <Subtitle>ขายแล้ว 125 ชิ้น</Subtitle>
+                    </ContainerDetial>
+
+                    <ContainerDetial>
+                        <NameStore>ชื่อร้านค้า</NameStore>
+                        <Subtitle>Noon</Subtitle>
+                    </ContainerDetial>
+
+                    <ContainerDetial>
+                        <IdSeller>รหัสผู้ขาย</IdSeller>
+                        <Subtitle>6010504660</Subtitle>
+                    </ContainerDetial>
+
+                    <Price>255  บาท</Price>
+
+                    <ContainerDetial>
+                        <Shipping>การจัดส่ง</Shipping>
+                        <Subtitle>ฟรีค่าจัดส่ง</Subtitle>
+                    </ContainerDetial>
+
+                    <ContainerDetial>
+                        <Amount>จำนวน</Amount>
+                        <Container_PlusandMinus>
+                            <PlusandMinus></PlusandMinus>
```

Comment:
กรณีที่ node ไม่มี children node (ไม่มีลูก) ให้เขียนแบบปิดตัวเองเลยนะ อย่างอันนี้จาก 
 เปลี่ยนเป็น <PlusandMinus /> ได้นะ
```suggestion
                            <PlusandMinus />
```

(เรียกว่า self closing tag)